packages/mask/src/social-network-adaptor/twitter.com/collecting/identity.ts

```diff
@@ -91,6 +118,7 @@ export const IdentityProviderTwitter: Next.CollectingCapabilities.IdentityResolv
     recognized: creator.EmptyIdentityResolveProviderState(),
     start(cancel) {
         resolveLastRecognizedIdentityInner(this.recognized, cancel)
+        if (hasNativeAPI) resolveLastRecognizedIdentityMobileInner(this.recognized, cancel)
```

Comment:
You should use screen resolution or domain (`m.twitter`) instead of `hasNativeAPI`

Reply:
hello

Reply:
Thanks, I've replaced it by `isMobileTwitter`.